scania-digital-design-system/tegel
# docs(events): prefix change

**Describe pull-request**  
Changing prefix in events doc story

**Solving issue**  
Fixes: DTS-1693

**How to test**  
1. Go to Storybook (of file changes)
2. Check that sdds is renamed to tds
